config/prod.json

```diff
@@ -0,0 +1,8 @@
+{
+  "apiUrl": "https://consent.dsde-dev.broadinstitute.org/api",
+  "ontologyApiUrl": "https://consent-ontology.dsde-dev.broadinstitute.org/",
+  "clientId": "1087760099392-u18o6sqvb8uljmdltflohmd34h6hik9b.apps.googleusercontent.com",
```

Comment:
This will need to change for prod - its in the dev project now. Fine for this PR, but we need to remember when setting up the prod env.